frontend/src/components/GddSend/TransactionResultLink.vue

```diff
@@ -4,9 +4,17 @@
       <b-col>
         <b-card class="p-0 gradido-custom-background">
           <div class="p-4">
-            <div class="h3 mb-5">{{ $t('gdd_per_link.created') }}</div>
+            <div class="h3">{{ $t('gdd_per_link.created') }}</div>
+          </div>
+          <b-row>
+            <b-col class="text-center">
+              <figure-qr-code :text="link" />
+            </b-col>
+          </b-row>
```

Comment:
this should be below the clipboard and only a button which shows the popup. Its not needed to have the full QR-Code visible here

Reply:
der code ist unterhalb des links. 
ich würde kein popup an dieser stelle machen weil hier der link generiert wird. es wäre für einen user ein unnötiger zusätzlicher click. 
ich habe den qr code etwas kleinr gemacht. 
ein user kann sofort seinen link kopieren oder das bild speichern. wir können jetzt nicht feststellen was für einen user relevant ist oder nicht.